cpu/fe310/cpu.c

```diff
@@ -45,37 +51,59 @@ void cpu_init(void)
  *  cmd_code = 3
  *  all other fields = 0
  */
-
-__attribute__ ((section (".ramfunc")))
-void init_flash(void)
+void flash_init(void)
 {
-    /* Update the QSPI interface to adjust to the CPU speed
-     * This function needs to execute from the RAM
-     * when the QSPI interface is being reconfigured because the flash
-     * can't be accessed during this time
+    /* In case we are executing from QSPI, (which is quite likely) we need to
+     * set the QSPI clock divider appropriately before boosting the clock
+     * frequency.
      */
-
-    /* Disable SPI flash mode */
-    SPI0_REG(SPI_REG_FCTRL) &= ~SPI_FCTRL_EN;
```

Comment:
Is it safe no ti disable?

Reply:
I don't know and this function was not used before anyway.